cmd/rook/userfacing/multus/validation/validation.go

```diff
@@ -94,21 +86,19 @@ func init() {
 
 	Cmd.AddCommand(runCmd)
 	Cmd.AddCommand(cleanupCmd)
+	Cmd.AddCommand(configCmd)
 
-	// flags on ALL subcommands - makes output more straightforward than using PersistentFlags() global flags on parent
-	for _, subCommand := range Cmd.Commands() {
-		// the default namespace is the current namespace the operator pod is running in if possible
-		defaultNamespace := os.Getenv(k8sutil.PodNamespaceEnvVar)
-		if defaultNamespace == "" {
-			defaultNamespace = DefaultValidationNamespace
-		}
-		subCommand.Flags().StringVarP(&validationConfig.Namespace, "namespace", "n", defaultNamespace,
```

Comment:
FYI, the behavior mentioned here (https://github.com/rook/rook/pull/12396/files#r1234438030) was moved from this section.